config/sync/core.extension.yml

```diff
@@ -95,7 +98,6 @@ module:
   twig_extensions: 0
   ultimate_cron: 0
   user: 0
-  views_custom_cache_tag: 0
```

Comment:
As above: I think we want to keep this module enabled: it's handy to provide more granular cache tags for views listings.